app/components/forms/user-payment-info-form.js

```diff
@@ -0,0 +1,106 @@
+import Component from '@ember/component';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { validPhoneNumber } from 'open-event-frontend/utils/validators';
+import { pick } from 'lodash-es';
+
+export default Component.extend(FormMixin, {
+
+  async didInsertElement() {
+    this._super(...arguments);
+    let userBillingInfo = pick(this.authManager.currentUser, ['billingContactName', 'billingCity', 'billingPhone', 'company', 'billingTaxInfo', 'billingAddress', 'billingZipCode', 'billingAdditionalInfo']);
```

Comment:
Why store to the variable ? You can directly use within set

Reply:
Fixed it